pallets/gear-program/src/lib.rs

```diff
@@ -139,6 +140,9 @@ pub mod pallet {
         /// - `value`: balance to be transferred to the program once it's been resumed.
         ///
         /// - `ProgramResumed(H256)` in the case of success.
+        ///
+        /// TODO: unfortunatelly we cannot pass pages data in [PageBuf],
```

Comment:
TODO line must be comment, not doc string

Reply:
Ok done